### PR TITLE
8134 post processing settings are not reactive

### PR DESCRIPTION
**Pull request by SYBIOTE**
## Summary

the effects were being sourced from the global state -> PostProcessingeffectsstate, but we were not setting the state anywhere, therefore its always the default 
i replaced the source with the PostProcessingComponent instead, it works now

## References

closes #_insert number here_

## Checklist
- [ ] If this PR is still a WIP, convert to a draft
- [ ] When this PR is ready, mark it as "Ready for review"
- [ ] [ensure all checks pass](https://github.com/etherealengine/etherealengine/wiki/Testing-&-Contributing)
- [ ] Changes have been manually QA'd
- [ ] Changes reviewed by at least 2 approved reviewer

## QA Steps

_List any additional steps required to QA the changes of this PR, as well as any supplemental images or videos._

### CODE REVIEW

**Comment by SYBIOTE** [comment]
the other are working accordingly as well

File: packages/editor/src/components/properties/PostProcessingSettingsEditor.tsx [hunk 1]
```diff
@@ -275,11 +276,16 @@ export const PostProcessingSettingsEditor: EditorComponentType = (props) => {
     return value
   }
 
-  const setPropertyValue = (prop, val) => {
-    prop.set(val)
+  // trigger re-render - @todo find out why just setting the value doesn't trigger the reactor
+  // action: debounced the set property value
 
-    // trigger re-render - @todo find out why just setting the value doesnt trigger the reactor
+  const debouncedComposer = debounce(() => {
```
Comment (HexaField) [inline packages/editor/src/components/properties/PostProcessingSettingsEditor.tsx:282]: debouncedComposer should be defined in module scope, then we can reduce this to about 200ms

**Review by HexaField — CHANGES_REQUESTED** [review]

File: packages/editor/src/components/properties/PostProcessingSettingsEditor.tsx (hunk 1 above)
Comment (SYBIOTE) [reply packages/editor/src/components/properties/PostProcessingSettingsEditor.tsx:282]: So define the debounce and use it from where the actual implementation is ? 
I'm on it

File: packages/engine/src/renderer/functions/configureEffectComposer.ts [hunk 2]
```diff
@@ -45,10 +44,14 @@ import { SceneState } from '../../ecs/classes/Scene'
 import { getComponent } from '../../ecs/functions/ComponentFunctions'
 import { PostProcessingComponent } from '../../scene/components/PostProcessingComponent'
 import { EffectMap, EffectPropsSchema, Effects } from '../../scene/constants/PostProcessing'
-import { RendererState } from '../RendererState'
-import { EffectComposerWithSchema, EngineRenderer, PostProcessingSettingsState } from '../WebGLRendererSystem'
+import { EffectComposerWithSchema, EngineRenderer } from '../WebGLRendererSystem'
 import { changeRenderMode } from './changeRenderMode'
 
+// can use both as needed
+export const debouncedConfigureEffectComposer = debounce(() => {
```
Comment (HexaField) [inline packages/engine/src/renderer/functions/configureEffectComposer.ts:51]: we can keep the debounce function in the editor module.

**Review by HexaField — CHANGES_REQUESTED** [review]

File: packages/engine/src/renderer/functions/configureEffectComposer.ts (hunk 2 above)
Comment (SYBIOTE) [reply packages/engine/src/renderer/functions/configureEffectComposer.ts:51]: Editor.ts?

Comment (SYBIOTE) [reply packages/engine/src/renderer/functions/configureEffectComposer.ts:51]: if you mean as a dependency in package.json, its already there

Comment (HexaField) [reply packages/engine/src/renderer/functions/configureEffectComposer.ts:51]: in PostProcessingSettingsEditor.tsx

**Review by HexaField — APPROVED** [review]

File: packages/engine/src/renderer/functions/configureEffectComposer.ts [hunk 3]
```diff
@@ -156,9 +158,9 @@ export const configureEffectComposer = (remove?: boolean, camera: PerspectiveCam
       })
       effects.push(textureEffect)
     }
+    console.log('DEBUG effects', effects)
```
[the remark this answers is not in the document]
Comment (SYBIOTE) [reply packages/engine/src/renderer/functions/configureEffectComposer.ts:161]: yup

[the remark this answers is not in the document]
Comment (SYBIOTE) [reply packages/engine/src/renderer/functions/configureEffectComposer.ts]: done

**Review by HexaField — APPROVED** [review]

**Comment by SYBIOTE** [comment]
oh this was resolved? im closing the issue then

**Pull request merged**